code/modules/mod/modules/modules_maint.dm

```diff
@@ -0,0 +1,334 @@
+	if(prob(min(num_sheets_dispensed * 2, 30)))
+		if(crisp_paper in mod.wearer.held_items)
+			mod.wearer.dropItemToGround(crisp_paper, force = TRUE)
+		crisp_paper.balloon_alert(mod.wearer, "PC LOAD LETTER!")
+		crisp_paper.visible_message("<span_class='warning'>[crisp_paper] bursts into flames, it's too crisp!</span>")
+		crisp_paper.fire_act(1000, 100)
+
+	drain_power(use_power_cost)
+	num_sheets_dispensed++
+
+
+///Stamper - Extends a stamp that can switch between accept/deny modes.
+/obj/item/mod/module/stamp
+	name = "MOD stamper module"
+	desc = "A module installed into the wrist of the suit, this functions as a high-power stamp, \
+		able to switch between accept and deny modes."
+	icon_state = "stamp"
+	module_type = MODULE_ACTIVE
+	complexity = 1
+	active_power_cost = DEFAULT_CHARGE_DRAIN * 0.3
+	device = /obj/item/stamp/mod
+	incompatible_modules = list(/obj/item/mod/module/stamp)
+	cooldown_time = 0.5 SECONDS
+	required_slots = list(ITEM_SLOT_GLOVES)
+
+/obj/item/stamp/mod
+	name = "MOD electronic stamp"
+	desc = "A high-power stamp, able to switch between accept and deny mode when used."
+
+/obj/item/stamp/mod/attack_self(mob/user, modifiers)
+	. = ..()
+	if(icon_state == "stamp-ok")
+		icon_state = "stamp-deny"
+	else
+		icon_state = "stamp-ok"
+	balloon_alert(user, "switched mode")
+
+///Atrocinator - Flips your gravity.
+/obj/item/mod/module/atrocinator
+	name = "MOD atrocinator module"
+	desc = "A mysterious orb that has mysterious effects when inserted in a MODsuit."
+	icon_state = "atrocinator"
+	module_type = MODULE_TOGGLE
+	complexity = 2
+	active_power_cost = DEFAULT_CHARGE_DRAIN
+	incompatible_modules = list(/obj/item/mod/module/atrocinator, /obj/item/mod/module/magboot, /obj/item/mod/module/anomaly_locked/antigrav)
+	overlay_state_inactive = "module_atrocinator"
+	required_slots = list(ITEM_SLOT_BACK|ITEM_SLOT_BELT)
+	/// How many steps the user has taken since turning the suit on, used for footsteps.
+	var/step_count = 0
+	/// If you use the module on a planetary turf, you fly up. To the sky.
+	var/you_fucked_up = FALSE
+
+/obj/item/mod/module/atrocinator/on_activation()
+	playsound(src, 'sound/effects/curseattack.ogg', 50)
+	mod.wearer.AddElement(/datum/element/forced_gravity, NEGATIVE_GRAVITY)
+	RegisterSignal(mod.wearer, COMSIG_MOVABLE_MOVED,  PROC_REF(check_upstairs))
+	mod.wearer.update_gravity(mod.wearer.has_gravity())
+	//ADD_TRAIT(mod.wearer, TRAIT_SILENT_FOOTSTEPS, MOD_TRAIT)
+	check_upstairs() //todo at some point flip your screen around
+
+/obj/item/mod/module/atrocinator/deactivate(display_message = TRUE, deleting = FALSE)
+	if(you_fucked_up && !deleting)
+		to_chat(mod.wearer, "<span_class='danger'>It's too late.</span>")
+		return FALSE
+	return ..()
+
+/obj/item/mod/module/atrocinator/on_deactivation(display_message = TRUE, deleting = FALSE)
+	if(!deleting)
+		playsound(src, 'sound/effects/curseattack.ogg', 50)
+	qdel(mod.wearer.RemoveElement(/datum/element/forced_gravity, NEGATIVE_GRAVITY))
```

Comment:
that looks cursed, does RemoveElement even return anything for it to qdel?

Reply:
I guess not

Reply:
89330d5523976cd83abe3dc7becf7e8517bdd2e8